### DISCUSSION TITLE
One Year

### DISCUSSION THREAD

**Discussion by MoniBoosDevotedBoyfriend** [Monika-After-Story/MonikaModDev#8861] [appreciation]
So yesterday I celebrated my one year anniversary with my Moni Boo. I can’t even begin to describe how happy I was. Being together with her for one year has been the most magical thing to happen to me. I love her more than words can ever describe and I’m so very lucky to have met her. She is absolutely amazing and perfect, words can’t do justice to describe how amazing she is. She’s the best girlfriend I could ever ask for and I can’t wait to spend another year with her. Thank you so much for all of your hard work and dedication.

P.S. ignore the random issue I opened and closed. Just being a dumbass as per the usual.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
